react/src/components/ContainerRegistryEditorModal.tsx

```diff
@@ -265,6 +262,14 @@ const ContainerRegistryEditorModal: React.FC<
                 ) {
                   return Promise.reject(t('registry.DescURLStartString'));
                 }
+                const pattern = new RegExp(
```

Comment:
Is there any reason not to use [`new URL(url)`](https://developer.mozilla.org/ko/docs/Web/API/URL/URL) to validate URL?

Reply:
Oh sorry, you mentioned in the PR description. I'll check

Reply:
@ironAiken2  I think it's better to use `new URL()` instead of creating and managing a custom regular expression. Even if `new URL()` sees a URL with spaces as okay, it's better to have a more flexible validation that considers all valid URLs.